test/test_results.py

```diff
@@ -14,8 +14,6 @@
 from asv import results, runner, util
 import pytest
 
-from .tools import example_results
```

Comment:
Looks like this is needes to load the fixture, that's one of the failures in the CI.